content/src/main/content/jcr_root/apps/acs-commons/cloudservices/pwa/components/scripttag/header/header.html

```diff
@@ -0,0 +1,10 @@
+<sly data-sly-use.configuration="${'com.adobe.acs.commons.cloudservices.pwa.Configuration'}"
+     data-sly-test.ready="${configuration.ready}"
+     data-sly-test.path="${configuration.scopePath}">
+    <link rel="manifest" href="${path}.pwa.manifest.json" type="application/manifest+json" crossOrigin="use-credentials">
```

Comment:
So scope paths MUST reside at or under a page tree, that has a cq:conf that has a valid PWA Configuration, right? What happens if it does not? Does it gracefully fail?